openvdb/tree/RootNode.h

```diff
@@ -1641,6 +1642,20 @@ RootNode<ChildT>::nodeCount(std::vector<Index32> &vec) const
     vec[ChildNodeType::LEVEL] = sum;
 }
 
+template<typename ChildT>
+inline void
+RootNode<ChildT>::activeTileCount(std::vector<Index32>& vec) const
+{
+    assert(vec.size() > LEVEL);
+    Index32 sum = 0;
+    for (MapCIter i = mTable.begin(), e = mTable.end(); i != e; ++i) {
+        if (isChild(i)) {
+            getChild(i).activeTileCount(vec);
+        } else if (isTileOn(i)) ++sum;
+    }
+    vec[LEVEL] = 0;// zero tiles at root level
```

Comment:
what's the point of having an entry for the Root::LEVEL if it is always zero?

Reply:
@kmuseth Just to make sure the size of the vector corresponds to the height (or depth) of this tree. To be consistent, I guess. Not sure 🙂